hypervisor/dm/vpci/pci_pt.c

```diff
@@ -294,9 +294,7 @@ void vdev_pt_write_vbar(struct pci_vdev *vdev, uint32_t idx, uint32_t val)
 
 	if (is_pci_io_bar(vbar->bar_type.bits)) {
 		vpci_update_one_vbar(vdev, idx, val, vdev_pt_allow_io_vbar, vdev_pt_deny_io_vbar);
-	}
-
-	if (is_pci_mem32_bar(vbar->bar_type.bits) || is_pci_mem64_bar(vbar->bar_type.bits) || vbar->is_mem64hi) {
+	} else if (is_pci_mem32_bar(vbar->bar_type.bits) || is_pci_mem64_bar(vbar->bar_type.bits) || vbar->is_mem64hi) {
```

Comment:
use  is_mem_bar()

Reply:
The is_mem_bar() only check bits[0]==0, For reserved mem type, should ignore write this vbar. Have tryed use is_mem_bar(), and SOS can't boot

Reply:
That is a former issue, it can boot SOS now, will use is_mem_bar()